packages/livekit-rtc/tsconfig.json

```diff
@@ -2,7 +2,16 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "dist",
-    "declarationDir": "dist"
+    "declarationDir": "dist",
+    "module": "ES2020",
+    "esModuleInterop": true,
+    "allowJs": true,
+    "declaration": true,
+    "declarationMap": true,
+    "target": "es2020",
+    "lib": ["es2020"],
+    "moduleResolution": "node",
+    "sourceMap": true,
```

Comment:
why do we need all these additions on the package level?

Reply:
this is a reverting artifact from before we added all of them at a workspace level, will remove